xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/transactions/SubmitMultiSignedResult.java

```diff
@@ -34,32 +34,33 @@
    * Text result code indicating the preliminary result of the transaction, for example "tesSUCCESS".
    *
    * @return An optionally-present {@link String} containing the result of the submission.
-   * @deprecated Use {@link #result()} instead.
    */
-  @Deprecated
```

Comment:
So, first -- apologies for the communications mixup here. However, the `main` branch is still on v2, so I don't think we can actually change the contracts here until we publish a more-final `v3`, and at the moment we're still keeping that branch v2-compatible until the very last moment (i.e., until most of #170 is complete in the `v3-dev` branch). See Discussion #187 for more details around the transition.

So, for now I think the only thing to be done here is to deprecate `result` with a message that it will transition to `engineResult` in v3.